roles/k3s/templates/smarter-device-manager-ds-jetson.yaml.j2

```diff
@@ -69,6 +69,11 @@ spec:
       nodeSelector:
         smarter-device-manager: enabled
         smarter-device-manager/configmap: jetson
+      tolerations:
+        - key: "{{ k3s_worker_taint.split('=')[0] }}"
+        operator: "Equal"
+        value: "{{ k3s_worker_taint.split('=')[1].split(':')[0] }}"
+        effect: "{{ k3s_worker_taint.split(':')[1] }}"
```

Comment:
It is difficult to maintain jinja templates containing logic.

I suggest putting lines into kolla/defaults.yml instead:
```
k3s_worker_taint_key:
k3s_worker_taint_value:
k3s_worker_taint_effect:
```

If necessary, you can also use those to template the string format, also in defaults.yml
```
# format like 'worker-node=true:NoSchedule'
k3s_worker_taint: "{{ k3s_worker_taint_key }}={{ k3s_worker_taint_value }}:{{ k3s_worker_taint_effect}}"
```